server.js

```diff
@@ -3406,10 +3406,10 @@ cache(function(data, match, sendBadge, request) {
   var allReleases = match[2];
   var format = match[3];
   let apiUrl = `/repos/${userRepo}/releases`;
-  var badgeData = getBadgeData('release', data);
   if (allReleases === undefined) {
     apiUrl = apiUrl + '/latest';
   }
+  var badgeData = getBadgeData('release', data);
```

Comment:
I'm unsure how this change is relevant to this pull request?

Reply:
I came across it because it’s in the middle of a github badge affected by #1861.